Tools/ManualValidationPipeline.cs

```diff
@@ -0,0 +1,3790 @@
+		public static string RemoteMainFolder = "//$remoteIP/";
+		public string SharedFolder = RemoteMainFolder+"/write";
+
+		public static string MainFolder = "C:\\ManVal";
+		public string runPath = MainFolder+"\\vm\\"; //VM working folder;
+		public string vmCounter = MainFolder+"\\vmcounter.txt";
+		public string VMversion = MainFolder+"\\VMversion.txt";
+		public string LogFile = MainFolder+"\\misc\\ApprovedPRs.txt";
+		public string PeriodicRunLog = MainFolder+"\\misc\\PeriodicRunLog.txt";
+		
+		public static string logsFolder = MainFolder+"\\logs"; //VM Logs folder;
+		public string timecardfile = logsFolder+"\\timecard.txt";
+		public string TrackerModeFile = logsFolder+"\\trackermode.txt";
+
+		public static string writeFolder = MainFolder+"\\write"; //Folder with write permissions;
+		public string SharedErrorFile = writeFolder+"\\err.txt";
+		public string StatusFile = writeFolder+"\\status.csv";
+
+		public static string ReposFolder = "C:\\repos\\"+repo;
+		public string DataFileName = ReposFolder+"\\Tools\\ManualValidationPipeline.csv";
+
+		public static string imagesFolder = MainFolder+"\\Images"; //VM Images folder;
+		public string Win10Folder = imagesFolder+"\\Win10-Created010424-Original";
+		public string Win11Folder = imagesFolder+"\\Win11-Created010424-Original";
+
+		public static string GitHubBaseUrl = "https://github.com/"+owner+"/"+repo;
+		public static string GitHubContentBaseUrl = "https://raw.githubusercontent.com/"+owner+"/"+repo;
+		public static string GitHubApiBaseUrl = "https://api.github.com/repos/"+owner+"/"+repo;
+		public string ADOMSBaseUrl = "https://dev.azure.com/ms";
+		
+		//ADOLogs - should be refactored to be in-memory.
+		public static string DestinationPath = MainFolder+"\\Installers";
+		public static string LogPath = DestinationPath+"\\InstallationVerificationLogs\\";
+		public static string ZipPath = DestinationPath+"\\InstallationVerificationLogs.zip";
+
+		public string CheckpointName = "Validation";
+		public string VMUserName = "user"; //Set to the internal username you're using in your VMs.;
+		public string GitHubUserName = "stephengillie";
+		//public string SystemRAM = (Get-CimInstance Win32_PhysicalMemory | Measure-Object -Property capacity -Sum).sum /1gb;
+
+		public int displayLine = 0;
+
+		public static string string_PRRegex = "[0-9]{5,6}";
+		public static string string_hashPRRegex = "[//]"+string_PRRegex;
+		public static string string_hashPRRegexEnd = string_hashPRRegex+"$";
+		public static string string_colonPRRegex = string_PRRegex+"[:]";
+		
+        public Regex regex_PRRegex = new Regex(@string_PRRegex);
+        public Regex regex_hashPRRegex = new Regex(@string_hashPRRegex);
+        public Regex regex_hashPRRegexEnd = new Regex(@string_hashPRRegexEnd);
+        public Regex regex_colonPRRegex = new Regex(@string_colonPRRegex);
+		
+		public string file_GitHubToken = "C:\\Users\\Stephen.Gillie\\Documents\\PowerShell\\ght.txt";

```

Comment:
This should reference the `USERPROFILE` environment variable instead of your specific user path

Reply:
Yes, thank you - this would be a good change to make during the refactor phase. 

Steps of lift-and-shift:

1. Folds correctly in N++
2. Compiles at all.
3. Function/module ouptuts at all.
4. Function/module outputs desired data. 

After this comes refactoring. 

I had been moving everything to step 4, and got about 1/3 of the way, but wasn't making much daily progress. So instead everything is currently at step 2 - just moved the last functions in the [3-27 nightly](https://github.com/microsoft/winget-pkgs/pull/131556/commits/5de9fbd878a056b9fe9133dd93af2d33b02d633a) commit last night. Now completing steps 3 and 4 involve building out some more connective functions to connect controls with their module, and testing to ensure output is correct. Hopefully, this will be complete by the end of next week.